### PR TITLE
Lph docs

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: src/app/modules/component-library/library/landing-page-header/overview/overview.component.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <section class="container-fluid">
   <div class="row">
     <div class="col">
-      <p class="mb-5 u-font-size-2">The landing page header is a configurable component that helps users identify where they are in an application and displays a collection of related objects. It may contain action items and navigation items.</p>
+      <p class="mb-4 u-font-size-2">The landing page header is a configurable component that helps users identify where they are in an application and displays a collection of related objects. It may contain action items and navigation items.</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/landing-page-header/overview/overview.component.html]: Do we really need .mb-4 here vs using the standard margin bottom for `<p>`? 
We should align with Catie about standardizing vertical spacing so we can move towards a spacing system.

File: src/app/modules/component-library/library/landing-page-header/usage/usage.component.html [hunk 2]
```diff
@@ -1,7 +1,122 @@
 <section class="container-fluid">
   <div class="row">
     <div class="col">
-      <p class="mb-5 u-font-size-2">The landing page header is often used on pages that contain data tables. It may contain a primary button or dropdown menu used to create an object, as well as tabbed navigation used to switch between data sets within a collection of related objects.</p>
+      <p class="mb-5 u-font-size-2">The page title is the only required element in a landing page header. All other additions shown below are optional and depend on the situation.</p>
+
+      <h2>Variations</h2>
+
+      <h3>Standard header</h3>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/landing-page-header/usage/usage.component.html]: We're inconsistent with sentence case vs title case in our headings (in buttons & links too). We should verify and update.

File: src/app/modules/component-library/library/landing-page-header/usage/usage.component.html [hunk 3]
```diff
@@ -1,7 +1,122 @@
...
+        <div class="card-body component-preview">
+          <bp-landing-page-header
+            headingText="Page Title"
+            [hasAction]="false"
+          ></bp-landing-page-header>
+        </div>
+      </div>
+      
+      <h3>Standard header with primary button</h3>
+      <p>Use this header for when a user can take an action, or a few actions, from the landing page itself. If there is more than one action on the page, keep in mind that there should still only be one primary button.</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/landing-page-header/usage/usage.component.html]: Can we get rid of 'for'? 
"Use this header when..."

File: src/app/modules/component-library/library/landing-page-header/usage/usage.component.html [hunk 4]
```diff
@@ -1,7 +1,122 @@
...
+                  <button class="tab" href="">Tab</button>
+                </div>
+              </div>
+            </div>
+          </bp-landing-page-header>
+        </div>
+      </div>
+
+      <h3>Standard header with primary button</h3>
+      <p>Use this header to navigate between related data sets and the user can take a global action, or actions</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/landing-page-header/usage/usage.component.html]: Add a period at the end of the sentence.

File: src/app/modules/component-library/library/landing-page-header/usage/usage.component.html [hunk 5]
```diff
@@ -1,7 +1,122 @@
...
+                  <button class="tab" href="">Tab</button>
+                </div>
+              </div>
+            </div>
+          </bp-landing-page-header>
+        </div>
+      </div>
+      
+      <h3>Standard header with dropdown menu</h3>
+      <p>Use this header to navigate between related data sets and there are more than three actions</p>
```
Comment (kaseyllabelle) [inline src/app/modules/component-library/library/landing-page-header/usage/usage.component.html]: We're missing a word (if? when?) after 'and'.
Add a period at the end of the sentence.

File: src/app/modules/component-library/library/landing-page-header/usage/usage.component.html (hunk 2 above)
Comment (APJohns) [reply src/app/modules/component-library/library/landing-page-header/usage/usage.component.html]: The mockups use sentence casing for the headings, buttons & links and lph align with the mockups.

**Pull request merged**